onvm/onvm_mgr/onvm_stats.c

```diff
@@ -408,26 +402,26 @@ onvm_stats_display_nfs(unsigned difftime, uint8_t verbosity_level) {
                 }
 
                 if (verbosity_level == ONVM_RAW_STATS_DUMP) {
-                        fprintf(stats_out, "%s,%u,%u,%" PRIu64 ",%" PRIu64 ",%" PRIu64 ",%" PRIu64 ",%" PRIu64
+                        fprintf(stats_out, "%s,%s,%u,%u,%u,%" PRIu64 ",%" PRIu64 ",%" PRIu64 ",%" PRIu64 ",%" PRIu64
```

Comment:
I suggest we define this String the same way we define ONVM_STATS_MSG and others, it takes up like 4 lines here and isn't very readable.

Reply:
well same with others obv